tools/releasing/deploy_staging_jars.sh

```diff
@@ -22,6 +22,14 @@
 ##
 MVN=${MVN:-mvn}
 CUSTOM_OPTIONS=${CUSTOM_OPTIONS:-}
+SUPPORTED_FLINK_VERSIONS=${SUPPORTED_FLINK_VERSIONS:-}
+
+# The variable SUPPORTED_FLINK_VERSIONS must be set, which contains all supported flink
+# versions with comma separated, e.g. "1.15,1.16.1.17".
+if [ "${SUPPORTED_FLINK_VERSIONS}" = "" ]; then
```

Comment:
Here we depend on a variable `SUPPORTED_FLINK_VERSIONS`. @lindong28 Could you help update the release doc[1] to set this variable before running the script?

[1] https://cwiki.apache.org/confluence/display/FLINK/Creating+a+Flink+ML+Release#CreatingaFlinkMLRelease-Createareleasebranch

Reply:
Sure. I will do it.